common/scylla_configure.py

```diff
@@ -46,6 +46,8 @@ class ScyllaMachineImageConfigurator:
         'post_configuration_script': '',
         'post_configuration_script_timeout': 600,  # seconds
         'start_scylla_on_first_boot': True,
+        'data_device': {'gce': 'attached',  # attached(default) - use attached devices nvme on gce and ebs on aws
```

Comment:
this is part of the product, users of GCP shouldn't know and bother about AWS... 
so as discussed it should be:
'data_device': 'attached' or 'instance_store'
you should set the default values in the SCT

Reply:
I added this so user don't know difference between gce or aws. It will automatically be choosed based on instance where it running if this parameter was not passed from sct or any other tool

Reply:
I agree with bentsi move this default handling to `common/scylla_create_devices`, if user didn't pass that down,
don't pass it down to `common/scylla_create_devices` and let it handle the defaults there.